build.ps1

```diff
@@ -119,6 +119,20 @@ Invoke-BuildStep 'Building NuGet.Clients projects - VS15 dependencies' {
 ## Building the VS14 Tooling solution
 Invoke-BuildStep 'Building NuGet.Clients projects - VS14 dependencies' {
         param($Configuration, $ReleaseLabel, $BuildNumber, $SkipRestore, $Fast)
+
+        if (-Not $SkipVS15)
```

Comment:
Fixed, but inverse of what you said since `(-not $SkipVS14) -and (-not $SkipVS15)` will be true when both are being built, which is the opposite of what `skip` needs.